resources/frameworks.yaml

```diff
@@ -191,7 +191,15 @@ TPOT:
 #    population_size: 25
 #    verbosity: 2
 
+####################################
+### TimeSeries AutoML frameworks ###
+####################################
 
+AutoGluonTS:
+  version: "stable"
+  description: |
+    AutoGluon-TimeSeries
+  project: https://auto.gluon.ai
```

Comment:
Ideally, we don't want to have a new framework if it supports only a specific kind of dataset. See my comment above.
If we start to have one framework for regression, one for classification, one for time series, one for anomaly detection and so on... then if becomes hard to compare "AutoML" frameworks.